glib/src/variant.rs

```diff
@@ -859,11 +859,32 @@ impl Variant {
     }
 
     // rustdoc-stripper-ignore-next
-    /// Variant has a container type.
+    /// Return whether this Variant is a container type.
     #[doc(alias = "g_variant_is_container")]
     pub fn is_container(&self) -> bool {
         unsafe { ffi::g_variant_is_container(self.to_glib_none().0) != ffi::GFALSE }
     }
+
+    // rustdoc-stripper-ignore-next
+    /// Return whether this Variant is in normal form.
+    #[doc(alias = "g_variant_is_normal_form")]
+    pub fn is_normal_form(&self) -> bool {
+        unsafe { ffi::g_variant_is_normal_form(self.to_glib_none().0) != ffi::GFALSE }
+    }
+
+    // rustdoc-stripper-ignore-next
+    /// Return whether input string is a valid `Variant::ObjectPath`.
+    #[doc(alias = "g_variant_is_object_path")]
+    pub fn is_object_path(string: &str) -> bool {
+        unsafe { ffi::g_variant_is_object_path(string.to_glib_none().0) != ffi::GFALSE }
+    }
+
+    // rustdoc-stripper-ignore-next
+    /// Return whether input string is a valid `Variant::Signature`.
+    #[doc(alias = "g_variant_is_signature")]
+    pub fn is_signature(string: &str) -> bool {
+        unsafe { ffi::g_variant_is_signature(string.to_glib_none().0) != ffi::GFALSE }
```

Comment:
```suggestion
        unsafe { from_glib(ffi::g_variant_is_signature(string.to_glib_none().0)) }
```

also for everything else here

Reply:
Indeed it seemed a bit odd, but I was following the existing code in `is_container()`. I'll take the liberty of tweaking that one too. Thanks for the prompt feedback!